src/Utilities/UnitTests/project.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
   "dependencies": {
     "xunit": "2.1.0",
+    "xunit.netcore.extensions": "1.0.0-prerelease-00116",
```

Comment:
Ahh . . . that makes sense. It's in the right place, then, since it's needed "everywhere".

`netstandard1.3` is a [platform standard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md) that implies running on .NET Core, full .NET Framework 4.6+, and Mono. We're primarily using it to indicate targeting .NET Core, though (hence my original comment).

---
In reply to: [59046481](https://github.com/Microsoft/msbuild/pull/560#discussion_r59046481) [](ancestors = 59046481)